src/actions/index.js

```diff
@@ -168,6 +168,7 @@ export const updateInfo = (field, value) => {
       return
     }
     // tracking field must be stored as string
+    // eslint-disable-next-line no-param-reassign
```

Comment:
After updating cozy-eslint, there were non-auto fixable errors. I prefer to simply disable the lines for now and not touch the runtime code